tap_stripe/__init__.py

```diff
@@ -725,6 +714,56 @@ def sync_sub_stream(sub_stream_name, parent_obj, updates=False):
                 Context.new_counts[sub_stream_name] += 1
 
 
+def sync_sub_stream(sub_stream_name, bookmark_value):
+    """
+    Get the parent records based on the bookmark and corresponding child stream records
+    """
+    stream_name = PARENT_STREAM_MAP[sub_stream_name]
+    stream_metadata = metadata.to_map(Context.get_catalog_entry(stream_name)['metadata'])
+
+    replication_key = metadata.get(stream_metadata, (), 'valid-replication-keys')[0]
+    # Invoice Items bookmarks on `date`, but queries on `created`
+    filter_key = replication_key
+
+    # Get bookmark for the stream
+    stream_bookmark = bookmark_value
```

Comment:
Any reason for this assignment if we are not changing the value of `bookmark_value`

Reply:
Removed extra assignment